jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MimeTypes.java

```diff
@@ -37,6 +39,18 @@
 public class MimeTypes
 {
     static final  Logger LOG = LoggerFactory.getLogger(MimeTypes.class);
+    private static final Set<Locale> KNOWN_LOCALES = Set.copyOf(Arrays.asList(Locale.getAvailableLocales()));
+
+    /**
+     * @param languageTag The language and optional country of the local. For example "en" or "en-GB".
+     * @return A {@link Locale} known to the JVM matching the languageTage or null if the Locale is not known;
```

Comment:
I'm not sure returning `null` is valid here.

All inputs for `languageTag` (that are not `null`), even empty `""`, dash `"-"`, and control codes will return the `und` `Locale` entry from the internal `Locale` cache (which is present in the `Locale.getAvailableLocales()` results too)

Reply:
@joakime see discussion above about quality lists